### PR TITLE
Issue#59 Client List View

### PR DESCRIPTION
**Pull request by TempestMia**
https://github.com/savoirfairelinux/santropol-feast/issues/59

Phone number column is populated with the home phone number as per the mockup we were shown - did not make use of the cell or work number.

selection_032

### CODE REVIEW

File: django/santropolFeast/member/models.py [hunk 1]
```diff
@@ -4,11 +4,16 @@
 
 import math
 
+HOME = 'Home phone'
```
Comment (maximest-pierre) [inline django/santropolFeast/member/models.py]: Can you implements the translation with _()?

File: django/santropolFeast/member/views.py [hunk 2]
```diff
@@ -8,17 +8,14 @@
...
     @method_decorator(login_required)
     def dispatch(self, *args, **kwargs):
         return super(ClientList, self).dispatch(*args, **kwargs)
 
-    def get_queryset(self):
-        # Here you need to return the list of clients
-        return 0
-
     def get_context_data(self, **kwargs):
         context = super(ClientList, self).get_context_data(**kwargs)
```
Comment (maximest-pierre) [inline django/santropolFeast/member/views.py]: Can you delete this method too?

**Comment by maximest-pierre** [comment]
Thank you but can you also remove your editor configuration file from the commit.

**Comment by manumilou** [comment]
I deleted the .idea directory in the dev branch. Thx

**Comment by TempestMia** [comment]
Thanks, sorry about that! Late night coding sometimes, Zzzz... Was going to
do it in the morning if you hadn't :)

On Thu, Apr 21, 2016 at 9:39 PM, Emmanuel Milou <notifications@github.com>
wrote:

> I deleted the .idea directory in the dev branch. Thx
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/savoirfairelinux/santropol-feast/pull/68#issuecomment-213203097>
>



-- 
Mia Z~

"One of the nice things about the internet is that you can write anything
you want, and then attribute it to a great mind."  -Oscar Wilde

**Pull request merged**